subprojects/docs/docs.gradle

```diff
@@ -402,6 +402,30 @@ samples {
                 from(sampleDirectory.dir("common"))
             }
             // TODO (donat) sanity checks probably don't do anything for samples located one level down in the gradle/kotlin directories
+        }*/
+
+        compositeBuildsBasics {
+            sampleDirectory = file("src/samples/build-organization/compositeBuilds/basic")
+            description = "Defining and using a composite build"
+            category = "Build organization"
+        }
+
+        compositeBuildsDeclaredSubstitutions {
+            sampleDirectory = file("src/samples/build-organization/compositeBuilds/declared-substitution")
+            description = "Declaring the dependencies substituted by an included build"
```

Comment:
This should describe what the use case of this sample is vs just which feature is being used

Reply:
I've added a new description that better explains the intent. WDYT?